static/scss/pages/settings.scss

```
@@ -0,0 +1,120 @@
+#settings-main {
+
+    .c-settings-headline {
+        @include text-title-2xs;
+        padding: $spacing-lg $spacing-lg $spacing-md ;
```

nit: ⚡ this line! (Channeling @groovecoder) 

```suggestion
        padding: $spacing-lg $spacing-lg $spacing-md;
```

---

Heh, why doesn't stylelint have a rule we can turn on to catch this :) 3fb3187